protocols/client/client.go

```diff
@@ -147,7 +147,10 @@ func agentDialer(addr *url.URL, enableYamux bool) dialer {
 		}()
 
 		var session *yamux.Session
-		session, err = yamux.Client(conn, nil)
+		sessionConfig := yamux.DefaultConfig()
+		// Disable keepAlive since we don't know how much time a container can be paused
+		sessionConfig.EnableKeepAlive = false
+		session, err = yamux.Client(conn, sessionConfig)
```

Comment:
As you are now passing config to the client api, you might want to set `sessionConfig.LogOutput`, similar to #289, so that any yamux warnings+errors are logged to the agent log?

Reply:
Hi, Jodh

Here the yamux client is running in the buildin proxy, instead of agent context, and there is no logger instance here.

Reply:
ah - ok. So do you not get any agent logs then?

Reply:
No, this is consumed by the runtime instead.